src/views/PositionPdfPlot/PositionPdfPlotWidget.tsx

```diff
@@ -30,12 +29,20 @@ type Props = {
 }
 
 type PanelProps = {
+    offscreenCanvas?: HTMLCanvasElement
 }
 
 const panelSpacing = 4
 
-const usePositionPdfDataModel = (fetchSegment: (q: FetchSegmentQuery) => Promise<number[][]>, numTimepoints: number, numPositions: number, segmentSize: number, multiscaleFactor: number) => {
+const usePositionPdfDataModel = (fetchSegment: (q: FetchSegmentQuery) => Promise<number[][]>, numPositions: number, segmentSize: number) => {
     const fetchSegmentCache = useFetchCache<FetchSegmentQuery, number[][]>(fetchSegment)
+    // The fetch cache cannot be memoized, because the 'get' depends on the state.
+    // Thus it is updated every time the cache updates, including asynchronously.
+    // Alternative: consider reworking this hook to A) be able to answer subset queries without
+    // triggering a new query (e.g. if we have range (60, 120), we can return range (60, 100) at
+    // the same downsample level without a new query) and B) using a callback to populate the
+    // data rather than returning a function that returns the data (so that there won't be a need
+    // to use a lookup-function update as the trigger for a rerender/data refresh).
```

Comment:
Just a few ideas of things we might want to do with the fetch cache. (It took me a bit to really get what it was doing.) This is not a high priority, but I wanted to get it down somewhere.